nyaa/views/account.py

```diff
@@ -223,6 +223,48 @@ def profile():
     return flask.render_template('profile.html', form=form)
 
 
+@bp.route('/trusted/request', methods=['GET', 'POST'])
+def request_trusted():
+    if not flask.g.user:
+        return flask.redirect(flask.url_for('account.login'))
+    trusted_form = None
+    deny_reasons = []
+    if flask.request.method == 'POST':
+        trusted_form = forms.TrustedForm(flask.request.form)
+        if trusted_form.validate():
+            ta = models.TrustedApplication()
+            ta.submitter_id = flask.g.user.id
+            ta.why_want = trusted_form.why_want_trusted.data
```

Comment:
I'd `.strip()` these two just because, although I forget if WTF passes the minimum length even if it's spaces.

Reply:
Do we get any issue if we don't strip them? In the very synthetic case of stripping them, it could break code blocks at the start of the field opened with 4 spaces. Obviously not something to worry about in this context though.

Reply:
`r.strip()`, then. It's simply a small useless abuse vector, pasting 4k spaces. 4k of data is not much to worry about, but hey, it's user input, and it's clean to `.rstrip()` it.